added askalcf doc (#963)
argonne-lcf/user-guides: docs/support/askalcf.md

<p align="center">
  <img src="./figures/rag.png" alt="Figure 1: AskALCF" width="500"/>
</p>
*Retrieval Augmented Generation Pipeline: LLM retrieves relevant external documents before generating an answer, grounding its output in factual, up-to-date information.*

## Knowledge Base

Below are the documents curated in the vector database used by AskALCF ChatBot.

<center>

| Data                              | Type / format |
|-----------------------------------|---------------|
| ALCF User Guide               | Markdown      |
| OLCF User Guide             | RST           |
| LLNL User Guide            | Web           |
| PBS Documentation         | PDF           |
| Slurm Documentation        | Web           |
| CUDA C Programming Guide    | Web           |
| SYCL Documentation         | PDF           |
| AMD HIP Documentation        | Web           |
| Intel OneAPI Documentation  | Web           |

> </center>

Reviewer: the `<center>` tags arent needed here

<p align="center">
  <img src="./figures/chatbot.png" alt="Figure 1: AskALCF" width="500"/>
</p>

## Example Questions

Please make your question as specific as possible. This helps the ChatBot retrieve the most relevant information and provide a high-quality answer.
- On Aurora, what is the command to log in, and what is the default quota for user home directories?
- How do I request 8 Aurora nodes all within the same rack using PBS? Script: pbs_submit_script.sh, Project: RackTest, Queue: prod, Duration: 1 hour, Filesystem: grand.
- Will my ALCF HPSS home directory remain accessible for my group members after my account termination?
- What is the best environment variable setting for oneCCL?
- How should I use copper to run a Python job at scale on Aurora?
- What is the optimal CPU binding on Aurora?

## Providing Feedback

* If you have feedback on a specific query, you can directly rate and comment on the chatbot’s response:
  - Click “Rate this assistant response”
  - Select a rating
  - Optionally provide additional comments
  - Click “Submit rating”

  We only track queries, responses, and ratings. No personal information is collected. 

> * For high-level comments or feedbacks, please contact: 📧 support@alcf.anl.gov

Reviewer: ```suggestion
* For high-level comments or feedback, please contact: [support@alcf.anl.gov](mailto:support@alcf.anl.gov)
```

> * All feedbacks are annoymouse, if you would like  to receive follow up to your feedbacks, please provide your contact info. 

Reviewer: ```suggestion
* All feedback is anonymous, by default. If you would like to receive follow-up communication, please provide your contact info. 
```
